test/verify/check-machines-virsh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python2
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import parent
+import unittest
+
+from testlib import *
+import machineslib
+
+
+class TestMachinesVirsh(machineslib.TestMachines):
+    def removeLibvirtDBus(self):
+        m = self.machine
+        if ("debian" in self.machine.image) or ("ubuntu" in self.machine.image):
+            m.execute("dpkg --remove libvirt-dbus || true")
+        else:
+            m.execute("rpm -e libvirt-dbus || true")
+
+    def setUp(self):
+        MachineCase.setUp(self)
+        self.startLibvirt()
+        self.removeLibvirtDBus()
+
+        # enforce use of cockpit-machines instead of cockpit-machines-ovirt
+        m = self.machine
+        m.execute("sed -i 's/\"priority\".*$/\"priority\": 100,/' {0}".format("/usr/share/cockpit/machines/manifest.json"))
+        m.execute("[ ! -e {0} ] || sed -i 's/\"priority\".*$/\"priority\": 0,/' {0}".format("/usr/share/cockpit/ovirt/manifest.json"))
```

Comment:
Right, so it's doing the exact same thing here, please move it to `TestMachines.setUp()` and call the `super` constructor in the two test cases.

Reply:
Done